app/components/privacy-module/app-details/geo-location/index.scss

```diff
@@ -0,0 +1,120 @@
+.map-container {
+  height: 35vw;
+  padding: 2.857em;
+}
+
+.map-body {
+  background-color: var(--privacy-module-geo-location-map-bg-color);
+}
+
+.map-header {
+  border-bottom: 1px solid var(--privacy-module-geo-location-map-border-color);
+  padding: 1.14em;
+}
+
+.drawer-body {
+  padding: 0 1.428em;
+  width: 642px;
+
+  .country-name {
+    margin: 1.285em 0;
+
+    .details-number {
+      margin-left: 0.285em;
+      border: 1px solid var(--privacy-module-geo-location-map-border-color);
+      border-radius: var(--privacy-module-geo-location-map-border-radius);
+      padding: 0.285em;
+      gap: 0.214em;
+      align-items: center;
+
+      .globe-icon {
+        font-size: 1.071rem !important;
+      }
+
+      .number {
+        font-size: 0.857rem;
+        font-weight: var(--privacy-module-geo-location-text-fw-bold);
+      }
+    }
+  }
+
+  .accordion-container {
+    .accordion {
+      border: 1px solid
+        var(--privacy-module-geo-location-accordion-border-color);
+      margin-bottom: 1.428em;
+    }
+
+    .accordion-header {
+      padding: 0.857em 0.714em 0.857em 1.428em;
+      border-bottom: 0px;
+      background-color: var(--privacy-module-geo-location-map-border-color);
+    }
+
+    .accordion-body {
+      padding: 1.428em;
+      gap: 1.07em;
+
+      .wrap-text {
+        overflow-wrap: anywhere;
+      }
+    }
+  }
+}
+
+.empty-container {
+  padding-top: 4em;
+  padding-bottom: 4em;
+  background-color: var(--privacy-module-geo-location-map-bg-color);
+  border-top: 1px solid var(--privacy-module-geo-location-map-border-color);
+
+  .header {
+    margin-top: 1.25em;
+  }
+
+  .body-text {
+    text-align: center;
+    max-width: 500px;
+  }
+}
+
+.country-tooltip {
+  z-index: 1000 !important;
+  box-shadow: 0px 1.88px 15.06px 0px rgba(75, 85, 110, 0.2) !important;
```

Comment:
If we cannot reuse the existing box shadows in `app/styles/_theme.scss` let's create a new special variable for this use case.